### ISSUE TITLE
Where are the Ruby communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/ruby#811]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Ruby supported by one or more large organizations?
* Does Ruby have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Ruby? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Ruby? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Ruby? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Ruby?
* Are there popular newsletters for Ruby?
* Is Ruby taught at programming bootcamps? (If so, what are some examples?)
* Is Ruby taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Ruby?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by brandondees** [comment]
It's not official, but https://www.rubyonrails.link/ is the biggest ruby community group I'm aware of. It's an open-to-public slack team full of friendly ruby devs from around the world.

**Comment by F3PiX** [comment]
GoRails has a quite lively Slack community, with Rails and Ruby devs.
In the Netherlands there is a Ruby Slack community with a Meetup Group

Euroku 2018 is coming up in Vienna, Austria in August, and is organised by members of a Ruby Vienna group.

**Comment by PatrickMcSweeny** [comment]
Do you know of specific communities (online or offline) that are enthusiastic about Ruby? 

[Ruby Together](https://rubytogether.org/)
[Columbus Ruby Brigade](http://columbusrb.com/)
[Portland Ruby Brigade](https://pdxruby.org/)
[Chicago Ruby](https://chicagoruby.org/)

Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Ruby?

[Rails Girls](http://railsgirls.com/) 

Are there popular newsletters for Ruby?

[Ruby Rogues Podcast](https://devchat.tv/ruby-rogues/)
